loader_hub/file/paged_csv/base.py

```diff
@@ -0,0 +1,39 @@
+"""Paged CSV reader.
+
+A parser for tabular data files.
+
+"""
+from pathlib import Path
+from typing import Any, Dict, List, Optional
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+
+
+class PagedCSVReader(BaseReader):
+    """Paged CSV parser.
+
+    Displayed each row in an LLM-friendly format on a separate document.
+    """
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
no need for `__init__` here

may be worth at some point to add a list of keys specifying order but don't need for now